protostar/commands/cairo1_commands/test_rust_command.py

```diff
@@ -0,0 +1,40 @@
+from argparse import Namespace
+from pathlib import Path
+from typing import Optional
+
+from protostar.cli import ProtostarArgument, ProtostarCommand
+from protostar.protostar_exception import ProtostarException
+
+from protostar.cairo.bindings import protostar_rust_bindings
+
+
+class TestRustCommand(ProtostarCommand):
+    @property
+    def name(self) -> str:
+        return "test-rust"
+
+    @property
+    def description(self) -> str:
+        return "Executes cairo1 tests in rust."
```

Comment:
```suggestion        
        return "Executes Cairo 1 tests in Rust."
```
A nitpick, but I just renamed everything in the docs to "Cairo 1" instead of "cairo1" etc. xD

Reply:
```suggestion
        return "Executes Cairo 1 tests with Rust implementation of Protostar."
```

Reply:
[address feeback etc](https://github.com/software-mansion/protostar/pull/1918/commits/0b1ef91db6dd4db747aa336b04f5241da132ee67)